gh-pages/doc/data-flow.html

```diff
@@ -0,0 +1,252 @@
+  <!DOCTYPE html>
```

Comment:
This directory is a mapping of the contents of the branch called gh-pages. (Which is the conventional name) 
I felt having two branches with no relation, was a bit confusing as you were expected to check both out in different clients and then copy between them. This allows both to exist within one client, and everyone automatically gets this same setup upon checkout. This allows us to see all the files at once and automate copying between them.